editor/filesystem_dock.cpp

```diff
@@ -1697,6 +1698,20 @@ void FileSystemDock::_move_operation_confirm(const String &p_to_path, bool p_ove
 	}
 }
 
+void FileSystemDock::_import_files_confirm(const PackedStringArray &p_files) {
+	Ref<DirAccess> dir = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
+
+	String &to_path = path;
+	for (int i = 0; i < p_files.size(); i++) {
+		String from = p_files[i];
+		String to = to_path.path_join(from.get_file());
+
+		dir->copy(from, to);
```

Comment:
This will overwrite existing files with the same name. Probably better to popup a confirm dialog, or skip it here and show a name collision report afterwards.

Reply:
Good catch, I'll make it so a confirm dialog appears :+1:

Reply:
Dialog has been added